### DISCUSSION TITLE
Double Negation Question

### DISCUSSION THREAD

**Discussion by nilandasd** [mongodb/mongoid#5825] [General]
What is expected when you use 2 not operators in sequence like `User.not.not.where(field: true)`? Shouldn't the query `User.where(foo: true)` be equivalent to `User.not.not.where(foo:true)`? I am finding that this is not the case, and am wondering if this usage of the not operator is unsupported or if this is a bug?

**Comment by jamis** [comment]
Hey @nilandasd -- looks like you might have found a bug! Seems like the fix ought to be straightforward ([this line here](https://github.com/mongodb/mongoid/blob/master/lib/mongoid/criteria/queryable/selectable.rb#L556)).

I've added a ticket to track this, here: https://jira.mongodb.org/browse/MONGOID-5780
